portfolio.html

```diff
@@ -139,6 +147,586 @@ <h2 class="description">I'm a software developer</h2>
+          <h3 class="project-header">Multi-Post Stories Gain+Glory</h3>
+          <ul class="project-language">
+            <li class="language ruby">Ruby on rails</li>
+            <li class="language css">css</li>
+            <li class="language js">JavaScript</li>
+            <li class="language html">html</li>
+          </ul>
+          <div class="button">
+            <button type="button">See Project</button>
+          </div>
+        </div>
+      </div>
+      <div class="projects-card">
+        <div class="image-holder"></div>
+        <div class="text-container">
+          <h3 class="project-header">Multi-Post Stories Gain+Glory</h3>
+          <ul class="project-language">
+            <li class="language ruby">Ruby on rails</li>
+            <li class="language css">css</li>
+            <li class="language js">JavaScript</li>
+            <li class="language html">html</li>
+          </ul>
+          <div class="button">
+            <button type="button">See Project</button>
+          </div>
+        </div>
+      </div>
+      <div class="projects-card">
+        <div class="image-holder"></div>
+        <div class="text-container">
+          <h3 class="project-header">Multi-Post Stories Gain+Glory</h3>
+          <ul class="project-language">
+            <li class="language ruby">Ruby on rails</li>
+            <li class="language css">css</li>
+            <li class="language js">JavaScript</li>
+            <li class="language html">html</li>
+          </ul>
+          <div class="button">
+            <button type="button">See Project</button>
+          </div>
+        </div>
+      </div>
+      <div class="projects-card">
+        <div class="image-holder"></div>
+        <div class="text-container">
+          <h3 class="project-header">Multi-Post Stories Gain+Glory</h3>
+          <ul class="project-language">
+            <li class="language ruby">Ruby on rails</li>
+            <li class="language css">css</li>
+            <li class="language js">JavaScript</li>
+            <li class="language html">html</li>
+          </ul>
+          <div class="button">
+            <button type="button">See Project</button>
+          </div>
+        </div>
+      </div>
+      <div class="projects-card">
+        <div class="image-holder"></div>
+        <div class="text-container">
+          <h3 class="project-header">Multi-Post Stories Gain+Glory</h3>
+          <ul class="project-language">
+            <li class="language ruby">Ruby on rails</li>
+            <li class="language css">css</li>
+            <li class="language js">JavaScript</li>
+            <li class="language html">html</li>
+          </ul>
+          <div class="button">
+            <button type="button">See Project</button>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+  <section class="about-section">
+    <div class="about-text">
+      <h2>class="primary-text">About me</h2> 
```

Comment:
- [ ] You have corrected the `hx` tag as suggested by the last code reviewer. Kindly correct this part of your code by adding the class as an attribute of the tag for a better user interface 👇🏽 

![image](https://user-images.githubusercontent.com/58642949/193429459-31766266-169d-40ca-bd3f-f0b4be45ccdd.png)